### PR TITLE
Fix #2614: Profile Chooser First Time A11y Fixes

### LINKED ISSUE #2614: [A11y] Profile Chooser First Time Flow
## Current Output

https://user-images.githubusercontent.com/9396084/106887527-48495a00-670b-11eb-8727-6b3b37917e29.mp4

## Issues Identified
- [ ] Talkback does not read the "Profile Image". Mark it as **not** importantForAccessibility.
- [ ] Talkback does not read the "+" image. Mark it as **not** importantForAccessibility.
- [ ] Mark Settings icon as **not** importantForAccessibility.

All these issues can be solved in one single PR.

These issues are same for mobile-landscape, mobile-portrait, tablet-landscape and tablet-portrait.

Note: To understand the above issue(s) completely, it is recommended that you setup Talkback and play with the app keeping it on and that will give you better context.

**Accessibility Guide**: https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #2614 

For this you will need to setup `A11y Scanner` and `Talkback` in your mobile. [Guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide)

## Output
https://user-images.githubusercontent.com/9396084/108260281-28447c80-7188-11eb-8d20-2a70b97765fe.mp4

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by Arjupta — APPROVED** [review]
LGTM

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -186,6 +186,7 @@
   </plurals>
   <string name="bar_separator">\u0020|\u0020</string>
   <!-- ProfileChooserFragment -->
+  <string name="profile_chooser_label">Profile selection page</string>
```
Comment (BenHenning) [inline app/src/main/res/values/strings.xml]: Perhaps instead: ``profile_chooser_activity_label`` to clarify this specifically pertains to an activity.

**Review by BenHenning — APPROVED** [review]
Thanks @rt4914! LGTM. Just had one comment.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/values/strings.xml]: Done. Resolving this now.

**Pull request merged**